joblib/_parallel_backends.py

```diff
@@ -253,7 +255,7 @@ def configure(self, n_jobs=1, parallel=None, **backend_args):
             # Avoid unnecessary overhead and use sequential backend instead.
             raise FallbackToBackend(SequentialBackend())
         self.parallel = parallel
-        self._pool = ThreadPool(n_jobs)
```

Comment:
Actually `_worker_processes` is a bad name because we have threads.  Maybe just `_workers` then,